R/dettl.R

```diff
@@ -31,6 +31,8 @@ DataImport <- R6::R6Class(
     load_ = NULL,
     extract_test_ = NULL,
     transform_test_ = NULL,
+    extract_passed = FALSE,
```

Comment:
What happens at present if the user does extract, transforms, then re-extracts, does `extract_passed` get re-FALSED (and the data get nulled?)

Reply:
We don't ever do that. We could have the `reload` function on the data import object reset these as currently that is how changes to sources can be used without rebuilding the object from scratch.